src/main/pg/rx_spi.c

```diff
@@ -31,7 +31,11 @@
 
 #include "rx/rx_spi.h"
 
-PG_REGISTER_WITH_RESET_FN(rxSpiConfig_t, rxSpiConfig, PG_RX_SPI_CONFIG, 0);
+#ifndef RX_SPI_LED_PIN
+#define RX_SPI_LED_PIN NONE
+#endif
+
+PG_REGISTER_WITH_RESET_FN(rxSpiConfig_t, rxSpiConfig, PG_RX_SPI_CONFIG, 1);
```

Comment:
Incrementing the version isn't needed now - all we are doing is appending more fields to the end of the parameter group, which is fine without a version increment.

Reply:
Done.
I still need to understand it better 😄